packages/documentation/src/content/docs/telemetry/integrating.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Building on Top of Rafiki Metrics
```

Comment:
Maybe something like: Deploying Custom Telemetry Service? Since "Building on Top of Rafiki Metrics" sounds more like adding additional metrics